src/tchannel_bridge.js

```diff
@@ -19,27 +19,29 @@
 // THE SOFTWARE.
 
 import * as constants from './constants';
-import DefaultContext from '../src/default_context';
+import DefaultContext from './default_context';
 import Span from './span';
 import SpanContext from './span_context';
 import Utils from './util';
 import opentracing from 'opentracing';
-import Tracer from '../src/tracer';
-import TextMapCodec from '../src/propagators/text_map_codec';
+import Tracer from './tracer';
+import TextMapCodec from './propagators/text_map_codec';
 
 let TCHANNEL_TRACING_PREFIX = '$tracing$';
 
 export default class TChannelBridge {
     _tracer: Tracer;
     _codec: TextMapCodec;
+    _contextFactory: Function;
 
-    constructor(tracer: Tracer) {
+    constructor(tracer: Tracer, contextFactory: Function) {
```

Comment:
I would suggest going with `options` 2nd parameter and reading `contextFactoryFn` from it. It makes this bridge easier to extend in the future.